tasks/main.yml

```diff
@@ -1 +1,60 @@
 ---
+
+- name: Assert configuration
+  assert:
+    that:
+      - sansible_aws_vpc_cidr is not none and sansible_aws_vpc_cidr | length > 0
+      - sansible_aws_vpc_cidr_subnet_private_a and sansible_aws_vpc_cidr_subnet_private_a | length > 0
```

Comment:
Do we want to force always having to use AZ a?  What if I only want a single subnet it AZ c?

Reply:
Yeah this is intentional to make all the conditionals work in the template, if someone needs such a setup then we can revisit this.